package/greenpak-programmer/Config.in

```diff
@@ -0,0 +1,7 @@
+config BR2_PACKAGE_GREENPAK_PROGRAMMER
+	bool "greenpak_programmer - GreenPAK Programming Utility"
```

Comment:
We should just use a single word, package name, for the bool.  As shown in [the manual](https://nightly.buildroot.org/#writing-rules-config-in), although not entirely clear it is de-facto practice in Buildroot, so we should follow that here too.

In fairness, we've not been entirely consistent with this in other packages, but I plan to address this coming weeks.